language-support/java/codegen/src/main/scala/com/digitalasset/daml/lf/codegen/InterfaceTree.scala

```diff
@@ -124,7 +123,7 @@ private[codegen] object InterfaceTree extends StrictLogging {
 
   def fromInterface(interface: Interface): InterfaceTree = {
     val builder = new InterfaceTreeBuilder(new mutable.HashMap())
-    interface.getTypeDecls.asScala.foreach { case (identifier, typ) =>
+    interface.typeDecls.foreach { case (identifier, typ) =>
```

Comment:
The `getTypeDecls` implementation takes a Scala collection and converts it a Java one. So yeah...